File: pytools/examples/lua/server/protocols/AckRoleLoginOkNoRole.lua
module("AckRoleLoginOkNoRole", package.seeall)


function new()
	local tab_obj = {}
	setmetatable(tab_obj, {__index = AckRoleLoginOkNoRole})
	tab_obj:init()
	return tab_obj
end

function init(self)
end


function encode(self)
	pack = packet.new()
	return pack:encode(Msg.P_ACK_ROLE_LOGIN_OK_NO_ROLE)
end

File: pytools/examples/csharp/server/Protocol/AckRoleLoginOkNoRole.cs
using System.Collections;
using System.Collections.Generic;


public class AckRoleLoginOkNoRole
{


	public Packet Encode()
	{
		Packet packet = new Packet();
		packet.Encode(Msg.P_ACK_ROLE_LOGIN_OK_NO_ROLE);
		return packet;
	}



}
